### PR TITLE
feature/ignore-interface-in-privatize-constants-command

### PR DESCRIPTION
**Pull request by alexndlm**

### CODE REVIEW

File: src/Command/PrivatizeConstantsCommand.php [hunk 1]
```diff
@@ -64,7 +64,14 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $sources = (array) $input->getArgument('sources');
         $excludedPaths = (array) $input->getOption('exclude-path');
 
-        $phpFileInfos = PhpFilesFinder::find($sources, $excludedPaths);
+        $phpFileInfos = PhpFilesFinder::find(
+            $sources,
+            $excludedPaths,
+            static fn (SplFileInfo $splFileInfo): bool => ! str_ends_with(
+                $splFileInfo->getFilenameWithoutExtension(),
+                'Interface'
```
Comment (TomasVotruba) [inline src/Command/PrivatizeConstantsCommand.php:72]: What about interface that do not have such suffix?

Comment (alexndlm) [reply src/Command/PrivatizeConstantsCommand.php:72]: Sorry, do not get your idea.

Comment (TomasVotruba) [reply src/Command/PrivatizeConstantsCommand.php:72]: This interface should be skipped too, but the there is no "Interface" suffix:

```php
interface NotSuffixed
{
}
```

Comment (alexndlm) [reply src/Command/PrivatizeConstantsCommand.php:72]: Parse file content for this check will be ok?

Comment (TomasVotruba) [reply src/Command/PrivatizeConstantsCommand.php:72]: Indeed, that's the way.

Somewhere here maybe: https://github.com/rectorphp/swiss-knife/blob/b2c4ffe818c3fa329ba4df27e490c7812a8fddce/src/PHPStan/ClassConstantResultAnalyser.php#L39

Comment (alexndlm) [reply src/Command/PrivatizeConstantsCommand.php:72]: I have done it another way. It does not cover all edge cases, but it should work for 99% of cases.

**Comment by TomasVotruba** [comment]
Hi, I'm currently working on refactoring to php-parser approach: https://github.com/rectorphp/swiss-knife/pull/44
It will only check class constants, so it includes fix for your issue by design.

Thanks for your patience :+1:

**Pull request closed without merging**